### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by Onyxzium** [inferno-collection/Fire-Alarm#8]
If it try to go in the near of the pd i got this crash and the admin panel dont work i can only navigate with the mouse but no buttons work and i cannot go out of the panel

![2019-10-16 01_19_15-New Issue · inferno-collection_Fire-Alarm](https://user-images.githubusercontent.com/39564156/66876799-5ffb4d00-efb3-11e9-8c10-c5edf1c6184e.png)

**Comment by cm8263** [comment]
Hello,

Could you fill out the template please.

```
**Describe the bug**
A clear and concise description of what the bug is.

**Steps to reproduce**
Steps to reproduce the behavior.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Resource version**
What resource version are you running, and have you tried the latest version?

**Additional context**
Add any other context about the problem here.
```

**Comment by dhawton** [comment]
Also, it would be helpful to attach the CitizenFX.log file from your FiveM\FiveM.app directory, and the Memory Dump file from FiveM\FiveM.app\crashes would also be helpful in debugging crash reasons.

**Comment by cm8263** [comment]
1. Are you running any YMAPs at the PD station?
2. Have you tried adding pull stations in other locations and accessing them there?
3. Have you made any changes to the files?
4. Have you tried a fresh install?
5. Have you tried running the resource by itself?

**Comment by dhawton** [comment]
@Onyxzium still waiting on that crash dump, CitizenFX.Log or answers to the previous questions. Without it, there is no way to determine what is going on.

**Comment by cm8263** [comment]
If the issue persists please create another issue with the requested information.
